src/Containers/Pages/Login/Login.js

```diff
@@ -1,9 +1,40 @@
-import React from "react";
+import React, {useState} from 'react';
+import './Login.css';
+import logo from "./googleicon.png";
+import firebase from '../../../firebase'
+  
+//   import { config } from "./test-credentials";
 
 class Login extends React.Component {
-  render() {
-    return <div>Login</div>;
-  }
+
+    // not used for now
+    state = {
+        signedIn: false,
+        user: null,
+    }
+
+    googleSignIn () {
+        const googleAuthProvider = new firebase.auth.GoogleAuthProvider();
+        firebase.auth().signInWithPopup(googleAuthProvider);
+        console.log("Google Sign In Page");
+    }
+
+    helpNeeded () {
+        console.log("Help is needed");
```

Comment:
Instead of adding console.log try putting a comment like "TODO: specify what it is trying to do"